mitodl/hugo-course-publisher
# Color the download button blue with white text.

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://trello.com/c/B1mofw0l/92-pdf-page-full-color-design

#### What's this PR do?
Implements the full color design for the PDF page, which seems to only be the blue colored download button.

#### How should this be manually tested?
Run the site or visit the deploy preview, ensure that the look is right compared to the design.

#### Screenshots (if appropriate)
![Screen Shot 2020-04-03 at 2 28 04 PM](https://user-images.githubusercontent.com/12089658/78393358-9f484500-75b7-11ea-8194-49eaf9ed81e4.png)

![Screen Shot 2020-04-03 at 2 28 13 PM](https://user-images.githubusercontent.com/12089658/78393372-a2dbcc00-75b7-11ea-877d-4a83dd9949ba.png)
